README.md

```diff
@@ -11,7 +11,7 @@ Resources
 
 Documentation:
 
-https://symfony.com/doc/3.0/book/security.html
+https://symfony.com/doc/4.4/book/security.html
```

Comment:
This is wrong to me. The 4.4 doc does not document the ACL component anymore as it was moved ouf of the core: https://symfony.com/doc/4.4/security/acl.html. So this does not link to the doc of that component.

This should link to the bundle docs instead (at https://github.com/symfony/acl-bundle/blob/main/src/Resources/doc/index.rst) or directly to the acl page that talks about that bundle (and then in `/current` rather than in `/4.4`).

Reply:
:+1: for using the bundle docs link (the one from github). We'll remove ACL completely from the online docs

Reply:
I don't know what this should lead to.
For now, he directs to the page with the message: 

```
Warning: You are browsing the documentation for Symfony 3.0, which is no longer maintained. 
```